iOS Client/Store/MBStore.swift

```diff
@@ -96,51 +118,47 @@ class MBStore: NSObject {
         performDownload(clientFunction: client.getArticlesWithCompletion, persistentContainer: persistentContainer, deserializeFunc: MBArticle.deserialize, completion: completion)
     }
     
-    private func loadDevotions(completion: @escaping ([MBDevotion]?, Error?) -> Void) {
-        
+    func saveDevotions(devotions: [LoadedDevotion]) throws {
+        try self.save(devotions, "devotions")
+    }
+    
+    private func read<T: Codable>(fromPath: String, _ resource: T.Type) throws -> T {
```

Comment:
super minor, but if we want to get fancy I think we could name this parameter like `fromPath path:String` (can't remember the exact syntax) so callers get intuitive `fromPath` view of the param name but then internally this function can refer to it as just `path`, which makes more sense when calling urlPackage(forPath: path)